posthog/hogql/transforms/lazy_tables.py

```diff
@@ -62,6 +62,9 @@ def visit_lazy_join_type(self, node: ast.LazyJoinType):
     def visit_lazy_table_type(self, node: ast.TableType):
         self.found_lazy = True
 
+    def visit_field_type(self, node: ast.FieldType):
+        self.visit(node.table_type)
```

Comment:
Well, while this change will get this test to run: https://github.com/PostHog/posthog/blob/f58b3458b05dde55ac25ebf948cdf419aa2a11cb/posthog/hogql/transforms/test/test_lazy_tables.py#L138-L147

… it seems like this creates a condition where the visitor can find itself on a cyclical path where it never stops visiting. The funnel tests demonstrate this behavior if run with

```
py.test posthog/hogql_queries/insights/funnels/test/test_funnel.py -k test_funnel_cohort_breakdown 
```

Reply:
Resolved by ab22a1361a90cc988976c37d941ec388c77443ff!